pkg/aws/client.go

```diff
@@ -543,3 +544,9 @@ func (c *awsClient) DeleteAccessKeys(username string) error {
 	// Complete, deleted all accesskeys for `username`
 	return nil
 }
+
+// CheckRoleExists checks to see if an IAM role with the same name
+// already exists
+func (c *awsClient) CheckRoleExists(roleName string) (bool, error) {
+	return checkRoleExists(c, roleName)
```

Comment:
Out of curiosity, why not have the iamClient call directly here instead of the helper?

Reply:
Great question, I started writing the helper thing it was going to do more than it does.